README.md

```diff
@@ -2,50 +2,65 @@
 
 # cert-operator
 
```

Comment:
This is mostly formatting. Don't look at this file as long as this comment is here :)